src/main/java/step3/domain/Lotto.java

```diff
@@ -1,36 +1,52 @@
 package step3.domain;
 
 import java.util.*;
+import java.util.stream.Collectors;
 
 public class Lotto {
 	private final Set<LottoNumber> numbers;
 
 	private static final int LOTTO_NUMBER_LENGTH = 6;
 	private static final String LOTTO_PHRASES = "로또 번호는 중복되지 않는 6개의 숫자로 이루어져야 합니다.";
```

Comment:
```suggestion
	private static final int LOTTO_NUMBER_LENGTH = 6;
	private static final String LOTTO_PHRASES = "로또 번호는 중복되지 않는 6개의 숫자로 이루어져야 합니다.";
	
	private final Set<LottoNumber> numbers;
```
자바에서 변수 선언은 상수 -> 인스턴스 변수로 선언합니다 :)

Reply:
앗 수정한다고 했는데 제가 꼼꼼히 확인을 안했나보네요ㅠㅠ 감사합니다!